package.json

```diff
@@ -55,13 +60,22 @@
 		"@wordpress/core-data": "2.4.0",
 		"@wordpress/data": "4.6.0",
 		"@wordpress/data-controls": "1.0.0",
+		"@wordpress/date": "3.5.0",
+		"@wordpress/dom": "2.5.1",
+		"@wordpress/dom-ready": "2.5.0",
 		"@wordpress/edit-post": "3.5.0",
 		"@wordpress/editor": "9.4.0",
 		"@wordpress/element": "2.5.0",
+		"@wordpress/escape-html": "1.5.0",
+		"@wordpress/format-library": "1.9.1",
+		"@wordpress/hooks": "2.6.0",
+		"@wordpress/html-entities": "2.5.0",
+		"@wordpress/i18n": "3.6.1",
 		"@wordpress/notices": "1.5.0",
 		"@wordpress/nux": "3.4.0",
 		"@wordpress/plugins": "2.4.0",
 		"@wordpress/rich-text": "3.4.0",
+		"@wordpress/scripts": "5.0.1",
```

Comment:
This likely is not required and may be a contributing factor to difficulties with the build. I added lots of dependencies and whether or not they're used should be reviewed, both for inclusion and exclusiong.

Reply:
Shouldn't we be able to drop most of them, except for the ones we `import` from explicitly? Anything else should be ideally pulled in transitively, no?

Reply:
Yes, drop everything that isn't imported explicitly 👍